flume/doc/design/architecture.md

```diff
@@ -1 +1,77 @@
-TBD
+#Cygnus architecture
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/d4635d3a6ec4fd7f21cec57ddf2a610061c2501c